subprojects/core/src/integTest/groovy/org/gradle/api/tasks/ExecIntegrationTest.groovy

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.tasks
+
+import org.gradle.integtests.fixtures.AbstractConsoleFunctionalSpec
+import spock.lang.Issue
+
+class ExecIntegrationTest extends AbstractConsoleFunctionalSpec {
```

Comment:
`JavaExecIntegrationTest` already exist how should we name a test that covers all of them?